src/pages/homepage/Homepage.jsx

```diff
@@ -0,0 +1,17 @@
+import { Banner, Featured } from "../../components/homepage";
```

Comment:
You can create an import export index.js file in the components folder and if you use jsconfig.json then the path can be shortened to just components
Suggestion for future projects, if possible